common/lumens.d.ts

```diff
@@ -0,0 +1,15 @@
+export function getLumenBalance(horizonURL: any, accountId: any): Promise<any>;
```

Comment:
Could we avoid using `any` in these? Just curious because they might be too much to type out.

Reply:
ah this file was auto-generated using the `tsc --declaration` command. Do you mean instead of using `any` using their correct types?

Reply:
changed: https://github.com/stellar/dashboard/pull/204/commits/e4c7b25f9072d1aab99fe8cb301338af58669585
but could you still elaborate what you mean by `because they might be too much to type out.`, for my own understanding